docs/guides/deployment/overriding-job-variables.md

```diff
@@ -0,0 +1,174 @@
+# Deeper Dive: Overriding Work Pool Job Variables
+
+As you might have read in [Deploying Flows to Work Pools and Workers](/guides/prefect-deploy/) guide, there are two ways to deploy flows to work pools: using a `prefect.yaml` file or using the `.deploy()` method.
+
+**In both cases, you can override job variables on a work pool for a given deployment.**
+
+While exactly _which_ job variables are available to be overridden depend on the type of work pool you're using at a given time, this guide will explore some common patterns for overriding job variables in both deployment methods.
+
+## Background
+First of all, what do we mean by _"job variables"_?
+
+Let's use `env` as an example, because it's the only job variable that is available to configure for all work pool types.
+
+When you create or edit a work pool, you can specify a set of environment variables that will be set in the runtime environment of the flow run.
+
+For example, you might want a certain deployment to have the following environment variables available:
+```json
+{
+  "EXECUTION_ENV": "staging",
+  "MY_NOT_SO_SECRET_CONFIG": "some-value",
+}
+```
+
+So, as opposed to hardcoding these values into your work pool in the UI (so that they would apply to all deployments associated with that work pool), you can override these values on a _per-deployment basis_.
+
+Let's look at how to do that.
+
+## Setup
```

Comment:
```suggestion
## How to override job variables
```

Reply:
updated in [72147fb](https://github.com/PrefectHQ/prefect/pull/12033/commits/72147fb9aa10088c247bbca378fa0ba1059be314)